src/test/resources/wildcardimport3/input/org/anotherexample/Foo.java

```diff
@@ -0,0 +1,5 @@
+package org.anotherexample;
+
+public class Foo {
+    void fooMethod() { }
```

Comment:
Sorry for missing this one. The PR is failing because this method is not set to "public."

Reply:
Don't feel bad for missing this - this is what CI is here for. I've fixed it.